mission_control/navigator_launch/launch/master_shore.launch

```diff
@@ -1,3 +1,9 @@
 <launch>
```

Comment:
Wouldn't we just be adding files at that point, there's only 3 params that will (probably) never change. For the perception it makes sense because there are generally a lot of params that will change on the fly.